curriculum/challenges/english/25-front-end-development/lab-house-painting/66d6a7a3e1aa411e94bf2346.md

```diff
@@ -0,0 +1,532 @@
+const windows = new __helpers.CSSHelp(document).getStyle("#window-1, #window-2")?.getPropertyValue("width") || new __helpers.CSSHelp(document).getStyle("#window-2, #window-1")?.getPropertyValue("width");
+assert(window2 || windows);
+```
+
+You should target `#window-2` and set its `height` property.
+
+```js
+const window2 = new __helpers.CSSHelp(document).getStyle("#window-2")?.getPropertyValue("height");
+const windows = new __helpers.CSSHelp(document).getStyle("#window-1, #window-2")?.getPropertyValue("height") || new __helpers.CSSHelp(document).getStyle("#window-2, #window-1")?.getPropertyValue("height");
+assert(window2 || windows);
+```
+
+You should target `#window-2` and set its `border` property.
+
+```js
+const window2 = new __helpers.CSSHelp(document).getStyle("#window-2")?.getPropertyValue("border");
+const windows = new __helpers.CSSHelp(document).getStyle("#window-1, #window-2")?.getPropertyValue("border") || new __helpers.CSSHelp(document).getStyle("#window-2, #window-1")?.getPropertyValue("border");
+assert(window2 || windows);
+```
+
+You should target `#window-2` and set its `background-color` property.
+
+```js
+const window2 = new __helpers.CSSHelp(document).getStyle("#window-2")?.getPropertyValue("background-color");
+const windows = new __helpers.CSSHelp(document).getStyle("#window-1, #window-2")?.getPropertyValue("background-color") || new __helpers.CSSHelp(document).getStyle("#window-2, #window-1")?.getPropertyValue("background-color");
+assert(window2 || windows);
+```
+
+You should have a `#door` `div` within the `#house`.
+
+```js
+assert.exists(document.querySelector("div#house  div#door"));
+```
+
+`#door` should have the `position` property set to `absolute`.
+
+```js
+assert.equal(getComputedStyle(document.querySelector("#door"))?.getPropertyValue("position"), "absolute");
+```
+
+You should target `#door` and set its `width` property.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle("#door")?.getPropertyValue("width"));
+```
+
+You should target `#door` and set its `height` property.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle("#door")?.getPropertyValue("height"));
+```
+
+You should target `#door` and set its `border` property.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle("#door")?.getPropertyValue("border"));
+```
+
+You should target `#door` and set its `background-color` property.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle("#door")?.getPropertyValue("background-color"));
+```
+
+Your `#roof` should have the `top` property set to `0`.
+
+```js
+assert.equal(getComputedStyle(document.querySelector("#roof"))?.getPropertyValue("top"), "0px");
+```
+
+Your `#door` should be placed at the bottom of the house.
+
+```js
+const door = getComputedStyle(document.querySelector("#door"))
+const doorBottom = Number(door.getPropertyValue("bottom").replace("px", ""));
+assert.strictEqual(doorBottom, 0);
+```
+
+Your `#door` should not be outside the house borders.
```

Comment:
What does border mean here?
The test failed even when the door was inside the borders:

![image](https://github.com/user-attachments/assets/e304859d-7e87-43fe-bdae-2d3b34695920)

Reply:
I changed the hint. Now it should be helpful.